var/spack/repos/builtin/packages/r-convevol/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RConvevol(RPackage):
+    """Quantifies and assesses the significance of convergent evolution"""
+
+    homepage = "https://cran.r-project.org/package=convevol"
+    url      = "https://cran.r-project.org/src/contrib/convevol_1.3.tar.gz"
+
```

Comment:
could you add a list_url for https://cran.r-project.org/src/contrib/Archive/convevol/ ?
this will help when 1.3 is no longer the current version (otherwise spack won't find it)

Reply:
added